src/MSBuildLocator/VisualStudioLocationHelper.cs

```diff
@@ -43,21 +43,36 @@ internal static IList<VisualStudioInstance> GetInstances()
                     e.Next(1, instances, out fetched);
                     if (fetched <= 0) continue;
 
-                    var instance = instances[0];
-                    var state = ((ISetupInstance2) instance).GetState();
+                    var instance = (ISetupInstance2) instances[0];
+                    var state = instance.GetState();
```

Comment:
If you're making little changes like this, may as well change "var" to ISetupInstance2